packages/typespec-client-generator-core/src/decorators.ts

```diff
@@ -48,12 +48,23 @@ import { getLibraryName } from "./public-utils.js";
 import { getSdkEnum, getSdkModel, getSdkUnion } from "./types.js";
 
 export const namespace = "Azure.ClientGenerator.Core";
-const AllScopes = Symbol.for("@azure-core/typespec-client-generator-core/all-scopes");
+export const AllScopes = Symbol.for("@azure-core/typespec-client-generator-core/all-scopes");
```

Comment:
I would watch out what you export here without changing the `index.ts` this is now accessible to everyone consuming the package.

This means that things you might consider internal could be seen as breaking for someone that took dependency on it.

I would recommend being explicit in `index.ts` of what is exported that way this is not exposed. Or reorganize a little bit the files to be sure anything in decorators.ts is meant to be accesible outside of this package

Reply:
Moving `AllScopes` to `internal-utils` instead

Reply:
moved